### PR TITLE
Join 2019 Submission

### PR DESCRIPTION
**Pull request by DylanBaker**
Thought it might be easier to edit a file so moved [this issue](https://github.com/dbanalyticsco/Fonz/issues/3) across to this PR. 

(It's possible there is a way to convert the issue directly? I'm not super familiar with the workflow.)

### CODE REVIEW

File: docs/join2019_submisson.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#### Session Title
+
+### A Reliable Development Workflow with CI for Looker
```
Comment (DylanBaker) [inline docs/join2019_submisson.md]: I'm trying to figure out what angle is most sexy for this type of tool. 

What do you think of something like: `Develop LookML Like a Software Engineer: CI for Looker". I don't think the title is likely to make a huge difference, but I think we can probably make it pretty punchy.

Comment (DylanBaker) [reply docs/join2019_submisson.md]: Or even making it explicitly about the tool (which has a recognisable name)? 

"Introducing Fonz: a CI tool for reliable LookML and warehouse development"

File: docs/join2019_submisson.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+#### Session Abstract
+
+Occasionally, when we make changes to LookML or transform an underlying database table, we break downstream experiences in Looker by introducing database or content errors. These bugs are easy to deploy to production unknowingly, since they are runtime errors, not compiler errors. In short, they aren't discovered until a user runs a query. For little-used fields, these bugs can remain hidden for weeks or months.
+
+We believe in the power of CI for analytics, firmly of the view that analytics can adopt best practices from software engineering. We wanted a tool that would check for fields with database errors and content with LookML errors. We wanted to leverage continuous integration (CI) pipelines as part of LookML deployments or proposed database changes to provide a seamless development experience. Ultimately, we wanted to establish a baseline performance expectation for our Looker instance.
```
Comment (DylanBaker) [inline docs/join2019_submisson.md]: I've altered this slightly, making the second paragraph more explicitly about the values that drove the development of the tool. Let me know if you think that works. Happy to revert.

**Comment by DylanBaker** [comment]
(Also, definitely don't have to merge this in. Just thought it would be the easiest way to comment.)

File: docs/join2019_submisson.md (hunk 1 above)
Comment (joshtemple) [reply docs/join2019_submisson.md]: I like both of those a lot better. I kept thinking... "How do we _actually_ get people excited about attending a session on CI?"

I think the first one is slightly better. It feels a bit more provocative to me.

File: docs/join2019_submisson.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#### Session Title
+
+### Develop LookML Like a Software Engineer: Continuous Integration for Looker
+
+#### Session Abstract
+
+Occasionally, when we make changes to LookML or transform an underlying database table, we break downstream experiences in Looker by introducing database or content errors. These bugs are easy to deploy to production unknowingly, since they are runtime errors, not compiler errors. In short, they aren't discovered until a user runs a query. For little-used fields, these bugs can remain hidden for weeks or months.
+
+We wanted a tool that would check for fields with database errors and content with LookML errors. We firmly believe that analytics should adopt best practices from software engineering, and we're especially excited by the power of continuous integration (CI). We could run this tool in a CI pipeline before deploying to production or after database changes, helping to establish a baseline performance expectation for our Looker instance.
```
Comment (joshtemple) [inline docs/join2019_submisson.md]: I adjusted this paragraph slightly to make it flow a bit more clearly.

**Comment by joshtemple** [comment]
Updated the title and made a small change in the second paragraph. If it looks good to you, merge away! This closes #4.

**Comment by DylanBaker** [comment]
@joshtemple That's merged in. Do you want to send the proposal itself? I'm also happy to do so.

**Pull request merged**